codegen/specs/stytch_b2b/docs/discovery_intermediate_sessions/exchange.response.md

```diff
@@ -0,0 +1,13 @@
+Response fields beyond those defined in `ResponseBase`:
+
+- `member_id`: The UUID of the new Member that was created.
+
+- `session_token`: A secret token for a given Stytch Session. Read more about session_token in our Session management guide.
+
+- `session_jwt`: The JSON Web Token (JWT) for a given Stytch Session. Read more about session_token in our Session Management guide.
```

Comment:
smallest nit - in all the places we have session token + jwt the m in `Session management` mismatches on capitalization between the two